app/views/brexit_checker/_action_list.html.erb

```diff
@@ -4,7 +4,7 @@
       <p class="govuk-body brexit-checker__action_title">
         <% if action.title_url.present? %>
           <a
-            class="govuk-link"
+            class="govuk-link brexit-checker__action_link"
```

Comment:
Unsure if I should be doing this, basically added these not for any CSS but so I could test more easily. Not sure if that's terrible practice?

Ended up going with it because it seemed to help me parse what was going on when looking quickly through the view?

Reply:
Yeah, I think you anticipated my comment - I would avoid adding CSS classes specifically for tests if possible.

Could you use `".brexit-checker__action_title .govuk-link"` instead?

Reply:
I think this one is ok with `.brexit-checker__action_title .govuk-link`,
But I couldn't find much to distinguish a guidance link, without getting some of these action links too. 

They're also `govuk-link` classes within actions.
I think I added the other class to make the method of selecting both of these more similar.

Reply:
I think I'd rather avoid adding classes unless really necessary, as it's confusing to have classes with no associated styling. I'd probably value that over similar methods of selecting.

Reply:
A current action looks like this in the output, this one that has both an action and guidance link:
```
<div class="govuk-grid-row brexit-checker__action">
    <div class="brexit-checker__content-wrapper">
      <p class="govuk-body brexit-checker__action_title">
          <a class="govuk-link brexit-checker__action_link" href="https://www.gov.uk/guidance/using-personal-data-after-brexit#what-you-need-to-do-before-a-no-deal-brexit" data-module="track-click" data-track-action="Your business or organisation - 1.16 - Action" data-track-category="brexit-checker-results" data-track-label="https://www.gov.uk/guidance/using-personal-data-after-brexit#what-you-need-to-do-before-a-no-deal-brexit" data-ecommerce-row="" data-ecommerce-path="/guidance/using-personal-data-after-brexit">Read the guidance from the Information Commissioner's Office (ICO) to check if you need to change your contract to keep accessing personal data from the EU</a>
      </p>

        <p class="govuk-body">
          You will not be able to access personal data from the EU without the right contract.
        </p>

        <div class="govuk-body govuk-!-font-size-16">
          <p>
            More information:
          </p>
          <p>
            <a href="https://www.gov.uk/guidance/using-personal-data-after-brexit" class="govuk-link brexit-checker__guidance_link" data-module="track-click" data-track-action="Your business or organisation - 1.16 - Guidance" data-track-category="brexit-checker-results" data-track-label="https://www.gov.uk/guidance/using-personal-data-after-brexit" data-ecommerce-row="" data-ecommerce-path="/guidance/using-personal-data-after-brexit">Using personal data in your business or organisation if there’s no Brexit deal</a>
          </p>
        </div>
    </div>
  </div>
```

trying to work out the right selectors to get me directly to, but then distinguish between the two anchor tags if i remove `brexit-checker__guidance_link`

Reply:
Hm, do we need to distinguish between them? Can we just check that there is a link with href to the relevant action link, and a link with href to the relevant guidance?

Reply:
could do `.brexit-checker__action>div>p>a` for the first one and `.brexit-checker__action>div>div>p>a`, what'd you reckon @vanitabarrett ?